src/Uno.UI/Microsoft/UI/Xaml/Controls/TreeView/TreeViewList.cs

```diff
@@ -0,0 +1,712 @@
+
+			// Uno Workaround - waiting for #1453, remove this when fixed
+			var treeViewItem = itemContainer;
+			var targetNode = NodeFromContainer(treeViewItem);
+			treeViewItem.UpdateIndentation(targetNode.Depth);
+			treeViewItem.UpdateSelectionVisual(targetNode.SelectionState);
+			// workaround end
+
+			base.PrepareContainerForItemOverride(element, item);
+
+			if (selectionState != itemNode.SelectionState)
+			{
+				ListViewModel.UpdateSelection(itemNode, selectionState);
+			}
+		}
+
+		protected override DependencyObject GetContainerForItemOverride()
+		{
+			var targetItem = new TreeViewItem() { IsGeneratedContainer = true }; // Uno specific IsGeneratedContainer
+			return targetItem;
+		}
+
+		// IFrameworkElementOverrides
+		protected override void OnApplyTemplate()
+		{
+			if (!m_itemsSourceAttached)
+			{
+				ItemsSource = ListViewModel;
+				IsItemClickEnabled = true;
+				m_itemsSourceAttached = true;
+			}
+
+			base.OnApplyTemplate();
+		}
+
+		// IUIElementOverrides
+		protected override AutomationPeer OnCreateAutomationPeer() =>
+			new TreeViewListAutomationPeer(this);
+
+		internal string GetDropTargetDropEffect()
+		{
+			if (string.IsNullOrEmpty(m_dropTargetDropEffectString))
+			{
+				UpdateDropTargetDropEffect(true, false, null);
+			}
+			return m_dropTargetDropEffectString;
+		}
+
+		internal void SetDraggedOverItem(TreeViewItem newDraggedOverItem)
+		{
+			m_draggedOverItem = newDraggedOverItem;
+		}
+
+		internal void UpdateDropTargetDropEffect(bool forceUpdate, bool isLeaving, TreeViewItem keyboardReorderedContainer)
+		{
+			//Preserve old value of string in case it's needed.
+			string oldValue = m_dropTargetDropEffectString;
+
+			TreeViewItem dragItem = null;
+			AutomationPeer dragItemPeer = null;
+
+			if (keyboardReorderedContainer != null)
+			{
+				dragItem = keyboardReorderedContainer;
+			}
+			else
+			{
+				var listItem = ContainerFromItem(m_draggedOverItem);
+				if (listItem != null)
+				{
+					dragItem = (TreeViewItem)listItem;
+				}
+			}
+
+			if (dragItem != null)
+			{
+				var dragItemString = "";
+				var itemBeforeInsertPositionString = "";
+				var itemAfterInsertPositionString = "";
+				var draggedOverString = "";
+
+				dragItemPeer = FrameworkElementAutomationPeer.FromElement(dragItem);
+				if (dragItemPeer != null)
+				{
+					dragItemString = dragItemPeer.GetName(); //dragItemPeer.GetNameCore();
+				}
+
+				if (string.IsNullOrEmpty(dragItemString))
+				{
+					dragItemString = ResourceAccessor.GetLocalizedStringResource(
+						ResourceAccessor.SR_DefaultItemString);
+				}
+
+				if (isLeaving)
+				{
+					m_dropTargetDropEffectString = StringUtil.FormatString(
+						ResourceAccessor.GetLocalizedStringResource(ResourceAccessor.SR_CancelDraggingString),
+						dragItemString);
+				}
+				else
+				{
+					//unused variables in MUX source
+					//TreeViewItem itemAfterInsertPosition = null;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)